pkr_extractor/walk.c

```diff
@@ -67,7 +67,7 @@ bool LoadFiles(PKRFile *buffer, uint32_t numFiles){
 //Prints the name of all the files in a dir
 void ListDirFiles(PKRFile *files, uint32_t numFiles){
 	for(uint32_t i = 0; i<numFiles; i++)
-		printf("\t%s\n", &files[i]);
+		printf("\t%s\n", files[i].name);
```

Comment:
Should be `&files[i].name`.

Reply:
You're not really familiar with the ampersand operator, are you?

Reply:
@dracc enjoy the block